e2e/pushpull_test.go

```diff
@@ -145,7 +145,7 @@ func TestPushInstall(t *testing.T) {
 		cmd.Command = dockerCli.Command("app", "push", "--tag", ref, filepath.Join("testdata", "push-pull", "push-pull.dockerapp"))
 		icmd.RunCmd(cmd).Assert(t, icmd.Success)
 
-		cmd.Command = dockerCli.Command("app", "install", ref, "--name", t.Name())
+		cmd.Command = dockerCli.Command("app", "install", ref, "--pull", "--name", t.Name())
```

Comment:
I think `--pull` is going to be removed soon...

Reply:
There was a bug in `LookupOrPullBundle` where it would pull even if the parameter was set to false, once fixed I needed to explicitly tell `install` to pull.